client/src/components/Settings/SettingsInviteView.res

```diff
@@ -33,50 +35,43 @@ let view = (state: T.t): list<Html.html<AppTypes.msg>> => {
   let inviteform = {
     let submitBtn = {
       let btn = if state.loading {
-        list{Icons.fontAwesome("spinner"), Html.h3(list{}, list{Html.text("Loading")})}
+        list{Icons.fontAwesome("spinner"), Html.h3(list{Attrs.class'(%twc("m-0 pl-2.5"))}, list{Html.text("Loading")})}
       } else {
-        list{Html.h3(list{}, list{Html.text("Send invite")})}
+        list{Html.h3(list{Attrs.class'(%twc("m-0"))}, list{Html.text("Send invite")})}
       }
 
-      Html.button(
-        list{
-          Attrs.class'("submit-btn"),
-          Html.Attributes.disabled(state.loading),
```

Comment:
Is this attribute in the new version? I don't see it

Reply:
Sorry, I missed it when creating the component. Please let me know if I should change anything in the submitBtn component or if there is a better way to do it; I am not sure about it.

Reply:
This is good for now. As a later step, you can look at making it more like the Contributor page and reusing some of the components and types for buttons and loading state, but that's not necessary for this PR.

Reply:
Does the spinner work here? You should maybe test it out with a tunnel against production to see.

Reply:
It seems to be working. I'll try to figure out how to use a tunnel to make sure.

![dark-Dark-Google-Chrome-2022-11](https://user-images.githubusercontent.com/87861924/200935173-9e6105f2-2709-4a84-84f6-79bc699cc2f4.gif)

Reply:
Adding a doc about tunnels would also be useful, right now we just have this: https://github.com/darklang/dark/blob/main/docs/running-against-production.md

Reply:
My personal setup:

- run node http-server using: `http-server backend/static/ -b --cors -g`
- tunnel using cloudflare tunnels (much harder to setup than ngrok, but also very reliable)

Reply:
This is the result of testing it with a tunnel against production.

![ocean-Dark-Google-Chrome-2022-11](https://user-images.githubusercontent.com/87861924/201062423-66603e1d-1636-4530-b283-e1fcf4f71ff9.gif)